uber/model_checks.py

```diff
@@ -56,6 +56,18 @@ def dealer_description(group):
 
 
 @prereg_validation.Group
+def dealer_categories(group):
+    if group.tables and not group.categories:
```

Comment:
Bands get tables - so I **think** it makes more sense to use group.is_dealer unless it's somehow been used for something else.